app-backend/build.sbt

```diff
@@ -117,3 +118,16 @@ lazy val database = Project("database", file("database"))
   .settings({
    libraryDependencies ++= slickDependencies ++ dbDependencies ++ loggingDependencies ++ Seq(Dependencies.akkaHttpExtensions)
   })
+
+lazy val tiler = Project("tileserver", file("tileserver"))
+  .settings(commonSettings:_*)
```

Comment:
I think this needs a setting to set the assembly jar name:

`assemblyJarName in assembly := "rf-tileserver.jar"`

or something similar